addon/components/frost-button.js

```diff
@@ -227,12 +246,17 @@ export default Component.extend(PropTypeMixin, {
     }
   }),
 
+  // FIXME: jsdoc
```

Comment:
I've found that enforcing something like this tends to shut off people's brains and I'd much rather have people be conscious of how their code is going to be read.  We're likely not going to come to an common agreement on this one, but here is my shot at it:

1.  When everything has comments, there is too much noise in a file - code gets harder to read and important comments can be missed since they don't contrast from the rest of the file.  
2.  Even simple comments require maintenance or they get out of sync with the source and I've seen enough of this in my past to be wary.  
3.  Many comments are redundant and effectively feel like writing double the code (I've seen debates over a single extra character).  
4.  People tend to downplay the need to name variables/functions appropriately believing comments cover all sins.  
5.  It's yet another syntax for people new to the domain to learn - necessary for some cases yes, but easier to lookup in those cases than learn by heart.